libs/jump_portrait/src/jump_portrait/fetch.py

```diff
@@ -119,6 +120,52 @@ def get_jump_image(
     return result
 
 
+def get_jump_image_iter(metadata: pl.DataFrame, channel: List[str],
+                        site: List[str], correction:str='Orig',
+                        print_progress: bool=True,
+                        ) -> (pl.DataFrame, List[tuple]):
+    '''
+       Load jump image associated to metadata in a threaded fashion.
+        ----------
```

Comment:
Indentation issue, see https://github.com/scikit-image/scikit-image/blob/v0.24.0/skimage/measure/_moments.py#L376-L405 for an example.